docs/deploy/deployment-overview.md

```diff
@@ -0,0 +1,14 @@
+---
+description: Quorum deployment overview
+---
+
+# Deployment overview
+
+The GoQuorum Ethereum client is part of the [ConsenSys Quorum](https://consensys.net/quorum/) platform.
+The following are examples of Quorum deployments.
+
+![Quorum deployment examples](../images/deployment-examples.png)
```

Comment:
I know they're examples of deployments to show what GoQuorum can do, but I'm a little worried this image might be misunderstood as a preview for the content under "Deploy" and people will expect to find this content there.  For example, if I was a new customer looking at this, I would go "cool, I want to deploy GoQuorum as a sidechain...now what do I do?" and we don't mention sidechain anywhere else.  
The content we do talk about is in the other headers, not under deploy, which we use for installation. I could see people getting confused and possibly frustrated. The image feels better suited on the front page or on overview page of GoQuorum.  My suggestion is to make a "What can you do with GoQurorum?" page or something similar and then provide those examples but include links. 

On the image, under the header "Quorum in Enterprise", the text is formatted as suggestion rather than a description. 
Recommend: "Quorum Enterprise environment **uses** a 3 node network...." instead of "suggests" to match other 3.

Reply:
Thanks for the feedback! This is actually more of Quorum (not GoQuorum) deployment overview, and it's hard to find a proper place for it on the GoQuorum doc site. I'm also not sure where to link to, as the examples are generic. I ended up putting the diagram on a less prominent page at the end of the deploy section. I link twice to the Quorum website, so hopefully the reader will know to go there for next steps on Quorum enterprise/sidechain/BSN/etc.